### ISSUE TITLE
VIS runtime shows only a white screen with vis2-2.0.22

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.vis#677]
**Describe the bug**  
After updating vis2 to version 2.0.22 no widget is shown in the vis runtinme. Only a white screen is visible.

**To Reproduce**  
1. update vis2 to version 2.0.22
2. create a new project and add a single widget e.g. basic - string (unescaped)
3. start runtime
4. vis runtime is only a white screen

**Expected behavior**  
Widget is shown in vis runtime

**Versions:**  
 - Adapter version: 
 - vis2 - v2.0.22
 - Vis 2 Jaeger Styled Widgets: 0.3.0
 - vis-2-widgets-energy: 0.1.10
 - vis-2-widgets-gauges: 0.1.11
 - vis-2-widgets-material: 0.2.12

**Screenshots & Logfiles**  

```
iobroker.vis-2-beta@2.0.22 using theme "colored" bootstrap.jsx:20:8

██╗ ██████╗ ██████╗ ██████╗  ██████╗ ██╗  ██╗███████╗██████╗ 
██║██╔═══██╗██╔══██╗██╔══██╗██╔═══██╗██║ ██╔╝██╔════╝██╔══██╗
██║██║   ██║██████╔╝██████╔╝██║   ██║█████╔╝ █████╗  ██████╔╝
██║██║   ██║██╔══██╗██╔══██╗██║   ██║██╔═██╗ ██╔══╝  ██╔══██╗
██║╚██████╔╝██████╔╝██║  ██║╚██████╔╝██║  ██╗███████╗██║  ██║
╚═╝ ╚═════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
main.f6203d6c.js:657:11
Nice to see you here! :) Join our dev community here https://github.com/ioBroker/ioBroker or here https://github.com/iobroker-community-adapters main.f6203d6c.js:657:26
Help us to create open source project with reactJS! main.f6203d6c.js:657:186
See you :) main.f6203d6c.js:657:253
[2023-03-21T21:20:27.841Z] Try to connect socket.io.js:66:41
Version vis-history: 1.0.0 vis-2-beta:75:25
Version vis-jqui-mfd: 1.0.12 vis-2-beta:110:21
Version vis-map: 1.0.3 vis-2-beta:45:25
Metro version: "1.2.0" vis-2-beta:645:10
Version justgage: 1.0.2 justgage.js:115:21
Version trashschedule: 2.2.0 trashschedule.js:163:21
Version Info-Adapter-Widget: 0.0.9 info.js:29:12
Download the Vue Devtools extension for a better development experience:
https://github.com/vuejs/vue-devtools lib.min.js:43:9053
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html lib.min.js:43:9256
Version vis-materialdesign: 0.5.9 materialdesign.js:33:25
Uncaught (in promise) TypeError: t.update is not a function
    captureSession baseclient.ts:160
    _sendSessionUpdate hub.ts:460
    captureSession hub.ts:386
    init sdk.ts:200
    init sdk.ts:94
    _initializeSentry widgets.min.js:1
    initializeSentry widgets.min.js:1
baseclient.ts:160:14
Laden fehlgeschlagen für das <script> mit der Quelle "http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-jaeger-design/static/js/node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js". vis-2-beta:1:1
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  ChunkLoadError: Loading chunk node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-…r_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js)
    j customWidgets.js:3
    e customWidgets.js:1
    e customWidgets.js:1
    I customWidgets.js:1
    y customWidgets.js:2
    f customWidgets.js:1
    S customWidgets.js:2
    j customWidgets.js:2
    88411 customWidgets.js:2
    consumes customWidgets.js:2
    consumes customWidgets.js:2
    e customWidgets.js:1
    e customWidgets.js:1
    JaegerDesignLayout customWidgets.js:1
    d customWidgets.js:1
    ge visUtils.jsx:777
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage r.Z is not an object or null
    80534 src_JaegerDesignCameras_jsx.2f7b361d.chunk.js:6
    e customWidgets.js:1
    JaegerDesignCameras customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage p.Z is not an object or null
    75460 src_JaegerDesignDimmer_jsx.79f35b1d.chunk.js:6
    e customWidgets.js:1
    JaegerDesignDimmer customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage u.Z is not an object or null
    73879 src_JaegerDesignLayout_jsx-node_modules_hoist-non-react-statics_dist_hoist-non-react-statics_-2f012d.b7f77990.chunk.js:1
    e customWidgets.js:1
    JaegerDesignScene customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage l.Z is not an object or null
    53220 src_JaegerDesignNews_jsx.fe06ff3a.chunk.js:6
    e customWidgets.js:1
    JaegerDesignNews customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage d.Z is not an object or null
    56315 src_JaegerDesignPlayer_jsx.8d830604.chunk.js:6
    e customWidgets.js:1
    JaegerDesignPlayer customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage y.Z is not an object or null
    53396 src_JaegerDesignShutter_jsx.79fcc6c3.chunk.js:6
    e customWidgets.js:1
    JaegerDesignShutter customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage C.Z is not an object or null
    19738 src_JaegerDesignState_jsx.d71d932f.chunk.js:6
    e customWidgets.js:1
    JaegerDesignState customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-jaeger-design.0:  TypeError: class heritage t.Z is not an object or null
    57001 src_JaegerDesignThermostat_jsx-src_css_montserrat_css.0d7674dc.chunk.js:1
    e customWidgets.js:1
    JaegerDesignThermostat customWidgets.js:1
    ge visUtils.jsx:778
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-energy.0:  ChunkLoadError: Loading chunk node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-…r_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js)
    j customWidgets.js:3
    e customWidgets.js:1
    e customWidgets.js:1
    I customWidgets.js:1
    y customWidgets.js:2
    f customWidgets.js:1
    S customWidgets.js:2
    j customWidgets.js:2
    88411 customWidgets.js:2
    consumes customWidgets.js:2
    consumes customWidgets.js:2
    e customWidgets.js:1
    e customWidgets.js:1
    JaegerDesignLayout customWidgets.js:1
    d customWidgets.js:1
    ge visUtils.jsx:777
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-energy.0:  TypeError: class heritage o.Z is not an object or null
    33136 Consumption.jsx:24
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-energy.0:  TypeError: class heritage _.Z is not an object or null
    44411 IntervalSelector.jsx:26
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-energy.0:  TypeError: class heritage A.Z is not an object or null
    85165 Distribution.jsx:36
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-gauges.0:  ChunkLoadError: Loading chunk node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-…r_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js)
    j customWidgets.js:3
    e customWidgets.js:1
    e customWidgets.js:1
    I customWidgets.js:1
    y customWidgets.js:2
    f customWidgets.js:1
    S customWidgets.js:2
    j customWidgets.js:2
    88411 customWidgets.js:2
    consumes customWidgets.js:2
    consumes customWidgets.js:2
    e customWidgets.js:1
    e customWidgets.js:1
    JaegerDesignLayout customWidgets.js:1
    d customWidgets.js:1
    ge visUtils.jsx:777
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-gauges.0:  TypeError: class heritage M.Z is not an object or null
    33078 BatteryGauge.jsx:18
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-gauges.0:  TypeError: class heritage a.Z is not an object or null
    10439 WaterGauge.jsx:40
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  ChunkLoadError: Loading chunk node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-…r_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js)
    j customWidgets.js:3
    e customWidgets.js:1
    e customWidgets.js:1
    I customWidgets.js:1
    y customWidgets.js:2
    f customWidgets.js:1
    S customWidgets.js:2
    j customWidgets.js:2
    88411 customWidgets.js:2
    consumes customWidgets.js:2
    consumes customWidgets.js:2
    e customWidgets.js:1
    e customWidgets.js:1
    JaegerDesignLayout customWidgets.js:1
    d customWidgets.js:1
    ge visUtils.jsx:777
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage B.Z is not an object or null
    76362 Blinds.jsx:108
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage _.Z is not an object or null
    91814 Thermostat.jsx:94
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage _.Z is not an object or null
    14390 Actual.jsx:110
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage _.Z is not an object or null
    56255 Static.jsx:47
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage _.Z is not an object or null
    87969 Clock.jsx:60
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage u.Z is not an object or null
    89178 SimpleState.jsx:137
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage w.Z is not an object or null
    85076 ViewInWidget.jsx:18
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage O.Z is not an object or null
    55207 Camera.jsx:48
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage h.Z is not an object or null
    35445 Security.jsx:29
    Webpack 3
instrument.ts:129:35
Laden fehlgeschlagen für das <script> mit der Quelle "http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/static/js/vendors-node_modules_mui_material_Button_Button_js-node_modules_mui_material_Dialog_Dialog_js-54b5d4.b8c39d55.chunk.js". vis-2-beta:1:1
Cannot load widget system.adapter.fullcalendar.0:  ChunkLoadError: Loading chunk vendors-node_modules_mui_material_Button_Button_js-node_modules_mui_material_Dialog_Dialog_js-54b5d4 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/s…dules_mui_material_Dialog_Dialog_js-54b5d4.b8c39d55.chunk.js)
    j jsonp chunk loading:27
    e ensure chunk:6
    e ensure chunk:5
    FullCalendar container-entry:3
    v container-entry:13
    ge visUtils.jsx:777
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage g.Z is not an object or null
    21107 Player.jsx:99
    Webpack 3
instrument.ts:129:35
Laden fehlgeschlagen für das <script> mit der Quelle "http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/static/js/vendors-node_modules_mui_x-date-pickers_TimePicker_TimePicker_js.177836ca.chunk.js". vis-2-beta:1:1
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage j.Z is not an object or null
    19050 Map.jsx:146
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.vis-2-widgets-material.0:  TypeError: class heritage o.Z is not an object or null
    47996 Html.jsx:6
    Webpack 3
instrument.ts:129:35
Cannot load widget system.adapter.scheduler.0:  ChunkLoadError: Loading chunk node_modules_iobroker_vis-2-widgets-react-dev_index_jsx-_adb40 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/vis-2-widgets-…r_vis-2-widgets-react-dev_index_jsx-_adb40.af5baa26.chunk.js)
    j customWidgets.js:3
    e customWidgets.js:1
    e customWidgets.js:1
    I customWidgets.js:1
    y customWidgets.js:2
    f customWidgets.js:1
    S customWidgets.js:2
    j customWidgets.js:2
    88411 customWidgets.js:2
    consumes customWidgets.js:2
    consumes customWidgets.js:2
    e customWidgets.js:1
    e customWidgets.js:1
    JaegerDesignLayout customWidgets.js:1
    d customWidgets.js:1
    ge visUtils.jsx:777
instrument.ts:129:35
TypeError: can't convert undefined to object
    request main.f6203d6c.js:1046
    readFile main.f6203d6c.js:1046
    updateUserCss visEngine.jsx:1659
    render visEngine.jsx:1687
    e9 main.f6203d6c.js:1010
    D9 main.f6203d6c.js:1010
    wa main.f6203d6c.js:1010
    s9 main.f6203d6c.js:1010
    n7 main.f6203d6c.js:1010
    l9 main.f6203d6c.js:1010
    yr main.f6203d6c.js:1010
    Ht main.f6203d6c.js:1010
    d main.f6203d6c.js:1044
    E main.f6203d6c.js:1044
instrument.ts:129:35
Uncaught TypeError: can't convert undefined to object
    request main.f6203d6c.js:1046
    readFile main.f6203d6c.js:1046
    updateUserCss visEngine.jsx:1659
    render visEngine.jsx:1687
    e9 main.f6203d6c.js:1010
    D9 main.f6203d6c.js:1010
    wa main.f6203d6c.js:1010
    s9 main.f6203d6c.js:1010
    n7 main.f6203d6c.js:1010
    l9 main.f6203d6c.js:1010
    yr main.f6203d6c.js:1010
    Ht main.f6203d6c.js:1010
    d main.f6203d6c.js:1044
    E main.f6203d6c.js:1044
```

**Comment by Feuer-sturm** [comment]
I deleted all vis-2 widget adapters and installed them afterwards once again.

Vis 2 Jaeger Styled Widgets: 0.3.0
vis-2-widgets-energy: 0.1.10
vis-2-widgets-gauges: 0.1.11
vis-2-widgets-material: 0.2.12

The problem with the white screen in the runtime is the same but there are now only a few errors in the log:

```
iobroker.vis-2-beta@2.0.22 using theme "colored" bootstrap.jsx:20:8

██╗ ██████╗ ██████╗ ██████╗  ██████╗ ██╗  ██╗███████╗██████╗ 
██║██╔═══██╗██╔══██╗██╔══██╗██╔═══██╗██║ ██╔╝██╔════╝██╔══██╗
██║██║   ██║██████╔╝██████╔╝██║   ██║█████╔╝ █████╗  ██████╔╝
██║██║   ██║██╔══██╗██╔══██╗██║   ██║██╔═██╗ ██╔══╝  ██╔══██╗
██║╚██████╔╝██████╔╝██║  ██║╚██████╔╝██║  ██╗███████╗██║  ██║
╚═╝ ╚═════╝ ╚═════╝ ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
main.f6203d6c.js:657:11
Nice to see you here! :) Join our dev community here https://github.com/ioBroker/ioBroker or here https://github.com/iobroker-community-adapters main.f6203d6c.js:657:26
Help us to create open source project with reactJS! main.f6203d6c.js:657:186
See you :) main.f6203d6c.js:657:253
[2023-03-22T19:26:34.580Z] Try to connect socket.io.js:66:41
Version vis-history: 1.0.0 vis-2-beta:75:25
Version vis-jqui-mfd: 1.0.12 vis-2-beta:110:21
Version vis-map: 1.0.3 vis-2-beta:45:25
Metro version: "1.2.0" vis-2-beta:645:10
Version justgage: 1.0.2 justgage.js:115:21
Version trashschedule: 2.2.0 trashschedule.js:163:21
Version Info-Adapter-Widget: 0.0.9 info.js:29:12
Download the Vue Devtools extension for a better development experience:
https://github.com/vuejs/vue-devtools lib.min.js:43:9053
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html lib.min.js:43:9256
Version vis-materialdesign: 0.5.9 materialdesign.js:33:25
Uncaught (in promise) TypeError: t.update is not a function
    captureSession baseclient.ts:160
    _sendSessionUpdate hub.ts:460
    captureSession hub.ts:386
    init sdk.ts:200
    init sdk.ts:94
    _initializeSentry widgets.min.js:1
    initializeSentry widgets.min.js:1
baseclient.ts:160:14
Laden fehlgeschlagen für das <script> mit der Quelle "http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/static/js/vendors-node_modules_mui_material_Button_Button_js-node_modules_mui_material_Dialog_Dialog_js-54b5d4.b8c39d55.chunk.js". vis-2-beta:1:1
Cannot load widget system.adapter.fullcalendar.0:  ChunkLoadError: Loading chunk vendors-node_modules_mui_material_Button_Button_js-node_modules_mui_material_Dialog_Dialog_js-54b5d4 failed.
(error: http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/s…dules_mui_material_Dialog_Dialog_js-54b5d4.b8c39d55.chunk.js)
    j jsonp chunk loading:27
    e ensure chunk:6
    e ensure chunk:5
    FullCalendar container-entry:3
    v container-entry:13
    ge visUtils.jsx:777
instrument.ts:129:35
Laden fehlgeschlagen für das <script> mit der Quelle "http://192.168.178.77:8082/vis-2-beta/widgets/fullcalendar/static/js/vendors-node_modules_mui_x-date-pickers_TimePicker_TimePicker_js.177836ca.chunk.js". vis-2-beta:1:1
TypeError: can't convert undefined to object
    request main.f6203d6c.js:1046
    readFile main.f6203d6c.js:1046
    updateUserCss visEngine.jsx:1659
    render visEngine.jsx:1687
    e9 main.f6203d6c.js:1010
    D9 main.f6203d6c.js:1010
    wa main.f6203d6c.js:1010
    s9 main.f6203d6c.js:1010
    n7 main.f6203d6c.js:1010
    l9 main.f6203d6c.js:1010
    yr main.f6203d6c.js:1010
    Ht main.f6203d6c.js:1010
    d main.f6203d6c.js:1044
    E main.f6203d6c.js:1044
instrument.ts:129:35
Uncaught TypeError: can't convert undefined to object
    request main.f6203d6c.js:1046
    readFile main.f6203d6c.js:1046
    updateUserCss visEngine.jsx:1659
    render visEngine.jsx:1687
    e9 main.f6203d6c.js:1010
    D9 main.f6203d6c.js:1010
    wa main.f6203d6c.js:1010
    s9 main.f6203d6c.js:1010
    n7 main.f6203d6c.js:1010
    l9 main.f6203d6c.js:1010
    yr main.f6203d6c.js:1010
    Ht main.f6203d6c.js:1010
    d main.f6203d6c.js:1044
    E main.f6203d6c.js:1044
```

**Comment by GermanBluefox** [comment]
Please test with 2.0.23

**Comment by Feuer-sturm** [comment]
Is fixed with 2.0.23. Thanks.
